<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCreateField</class>
 <widget class="QDialog" name="QCreateField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create or edit field</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>826</width>
        <height>1036</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QGroupBox" name="groupBoxGeneral">
         <property name="title">
          <string>General Data</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>Name: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_Name"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSize">
            <property name="text">
             <string>Size (ha):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spBox_Size"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelRegNumber">
            <property name="text">
             <string>Register Number: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="le_RegNumber"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxCrops">
         <property name="title">
          <string>Crops</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="gbCrop1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QWidget" name="widget1" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QCheckBox" name="check1">
                  <property name="text">
                   <string>Inter Crop</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget6" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <layout class="QFormLayout" name="formLayout_13">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblInterCrop1_2">
                     <property name="text">
                      <string>Inter Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbInterCrop1_2"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_2" native="true">
                  <layout class="QFormLayout" name="formLayout_12">
                   <item row="1" column="0">
                    <widget class="QLabel" name="lblCrop1_2">
                     <property name="text">
                      <string>Main Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QComboBox" name="cbCrop1_2"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbCrop2">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="widget2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="check2">
                  <property name="text">
                   <string>Inter Crop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget7" native="true">
                  <layout class="QFormLayout" name="formLayout_4">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblInterCrop2">
                     <property name="text">
                      <string>Inter Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbInterCrop2"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_4" native="true">
                  <layout class="QFormLayout" name="formLayout_8">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblCrop2">
                     <property name="text">
                      <string>Main Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbCrop2"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbCrop3">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QWidget" name="widget3" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="check3">
                  <property name="text">
                   <string>Inter Crop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget8" native="true">
                  <layout class="QFormLayout" name="formLayout_5">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblInterCrop3">
                     <property name="text">
                      <string>Inter Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbInterCrop3"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_6" native="true">
                  <layout class="QFormLayout" name="formLayout_11">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblCrop3">
                     <property name="text">
                      <string>Main Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbCrop3"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbCrop4">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QWidget" name="widget4" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="check4">
                  <property name="text">
                   <string>Inter Crop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget9" native="true">
                  <layout class="QFormLayout" name="formLayout_6">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblInterCrop4">
                     <property name="text">
                      <string>Inter Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbInterCrop4"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_8" native="true">
                  <layout class="QFormLayout" name="formLayout_10">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblCrop4">
                     <property name="text">
                      <string>Main Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbCrop4"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbCrop5">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QWidget" name="widget5" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="check5">
                  <property name="text">
                   <string>Inter Crop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget10" native="true">
                  <layout class="QFormLayout" name="formLayout_7">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblInterCrop5">
                     <property name="text">
                      <string>Inter Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbInterCrop5"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_9" native="true">
                  <layout class="QFormLayout" name="formLayout_9">
                   <item row="0" column="0">
                    <widget class="QLabel" name="lblCrop5">
                     <property name="text">
                      <string>Main Crop: </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QComboBox" name="cbCrop5"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QCreateField</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QCreateField</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
